qa/integration-tests-webapps/src/test/java/org/camunda/bpm/AbstractWebIntegrationTest.java

```diff
@@ -36,9 +36,30 @@
  * @author Roman Smirnov
  *
  */
-public abstract class AbstractWebappIntegrationTest {
+public abstract class AbstractWebIntegrationTest {
 
-  private final static Logger LOGGER = Logger.getLogger(AbstractWebappIntegrationTest.class.getName());
+  protected String TASKLIST_PATH = "app/tasklist/default/";
+
+  protected String appUrl;
+  protected TestUtil testUtil;
+
+  protected TestProperties testProperties;
+
+  @Before
+  public void before() throws Exception {
+    testProperties = new TestProperties(48080);
+    testUtil = new TestUtil(testProperties);
+  }
+
+  protected String getWebappCtxPath() {
+    return testProperties.getStringProperty("http.ctx-path.webapp", null);
+  }
+
+  protected String getRestCtxPath() {
+    return testProperties.getStringProperty("http.ctx-path.rest", null);
+  }
+
+  private final static Logger LOGGER = Logger.getLogger(AbstractWebIntegrationTest.class.getName());
```

Comment:
@tasso94, just a codestyle question. Is there a specific reason why the class methods and variables order is mixed up?

Reply:
Good catch!

Reply:
Since it's just a codestyle problem, I'll refactor it myself and merge the changes to master.